src/providers/ModalConditionalsDisplayAndInfoProvider.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, createContext } from 'react'
+
+export const ModalConditionalsDisplayAndInfoContext = createContext()
```

Comment:
I don't understand what is this file?
Why did you merge the 2 contexts into one?
This is what we talked about that you should separate.

Reply:
The reason why I merged the two files is that all of the values in this file are pertaining to the modal--whether the values are about displaying the modal itself or what's on it. Moreover, on the modal, I am going to have specific buttons displayed (hence, the 'Display' part of the variable name) depending upon the location from where the modal is invoked. With that said, I was going to suggest that maybe I should add another context file for those conditionals about what will be on the modal.

Reply:
> whether the values are about displaying the modal itself or what's on it

These are 2 different things that require different states. If you need to access the same data throughout the app, use another provider.

Also, you already have most of these states in `MeatInfoProvider` which you are duplicating.

> I was going to suggest that maybe I should add another context file for those conditionals about what will be on the modal.

What do they depend on?

Reply:
> These are 2 different things that require different states. If you need to access the same data throughout the app, use another provider.

Ok, I will make another provider then.  And I deleted the MeatInfoProvider and moved some of those values to the ModalConditionalsDisplayAndInfoProvider

> What do they depend on?

If the meat item modal was opened from the cart, then the update button and the remove button will be displayed on the modal. 

And I'm pretty sure we talked about the cart component. We talked about that the cart component and the App component were valid places which I could access the local storage and make the computations for the cart orders.

Reply:
>Try to find a way for each component that brings up the modals to pass its own conditionals, instead of having a Context for them.

ok, that sounds like a step further down the road--opening the modal from the cart--from where I am right now. Do you want me to focus on building the navbar, the home page, the cart, and the functionality to add items to the cart next?

Reply:
because opening a meat item in a modal assumea at least that I have the menu component coded in.